src/identity.rs

```diff
@@ -221,13 +221,19 @@ impl Password {
         S2: Into<String>,
         S3: Into<String>,
     {
-        let url = auth_url.into_url()?;
-        // TODO: more robust logic?
-        let token_endpoint = if url.path().ends_with("/v3") {
+        let mut url = auth_url.into_url()?;
+
+        let _p = url
```

Comment:
very nit: usually just `_` is used for throw-away variables

Reply:
here's the problem: it _is_ a throwaway variable :(

All these segment:: methods return _self_. The compiler complains if I don't use it, so I thought about assigning to `throwaway`. Do you see a better way for making the compiler happy?